src/web/components/SlotBodyEnd.tsx

```diff
@@ -16,60 +22,77 @@ type Props = {
 };
 
 export const SlotBodyEnd = ({
+    isSignedIn,
+    countryCode,
     contentType,
     sectionName,
     shouldHideReaderRevenue,
     isMinuteArticle,
     isPaidContent,
     tags,
 }: Props) => {
-    // Putting together the request payload
-    const contributionsPayload = {
-        tracking: {
-            ophanPageId: window?.guardian.config.ophan.pageViewId,
-            ophanComponentId: 'ACQUISITIONS_EPIC',
-            platformId: 'GUARDIAN_WEB',
-            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-            abTestName: 'remote_epic_test',
-            abTestVariant: 'api',
-            referrerUrl: window?.location.origin + window?.location.pathname
-        },
-        localisation: {
-            countryCode: 'GB', // TODO: make this dynamic
-        },
-        targeting: {
-            contentType,
-            sectionName,
-            shouldHideReaderRevenue,
-            isMinuteArticle,
-            isPaidContent,
-            tags,
-        },
-    };
+    const [epicContent, setEpicContent] = useState<{ html: string; css: string}>();
 
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
-    const { data: responseBody, error } = useApi(endpointUrl, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(contributionsPayload),
-    });
+    useEffect(() => {
+        const callFetch = async () => {
+            // Putting together the request payload
+            const contributionsPayload = {
+                tracking: {
+                    ophanPageId: window?.guardian.config.ophan.pageViewId,
+                    ophanComponentId: 'ACQUISITIONS_EPIC',
+                    platformId: 'GUARDIAN_WEB',
+                    campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+                    abTestName: 'remote_epic_test',
+                    abTestVariant: 'api',
+                    referrerUrl: window?.location.origin + window?.location.pathname
+                },
+                localisation: {
+                    countryCode
+                },
+                targeting: {
+                    contentType,
+                    sectionName,
+                    shouldHideReaderRevenue,
+                    isMinuteArticle,
+                    isPaidContent,
+                    tags,
+                    showSupportMessaging: shouldShowSupportMessaging(),
+                    isRecurringContributor: isRecurringContributor(isSignedIn),
+                    lastOneOffContributionDate: getLastOneOffContributionDate(),
+                },
+            };
 
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'slot-body-end');
-        return null;
-    }
-
-    if (responseBody && responseBody.data) {
-        const { data } = responseBody;
+            const getEpicContent = () => {
```

Comment:
maybe look at using `useApi`? its in `/src/web/lib/api.tsx` 

Also this function doesnt need to belong to the react component, I think moving the logic outside could give some breathing room to the logic

Reply:
So it was my fault this got taken out. @andre1050 was using it before.

I was actually thinking about this last night and I realised I was wrong. You _can_ use `useApi` here to simplify this component - sorry!

Reply:
FINE - sorry, unintentional caps ;)

Reply:
Reverted back to `useApi` but we learned a little bit more about the intricacies of `useEffect`.